test/conftest.py

```
@@ -1,11 +1,20 @@
 from collections.abc import Iterator
+from contextlib import contextmanager
 from pathlib import Path
+from typing import Any
 
 import pytest
-from blib2to3.pgen2.parse import contextmanager
-from sensai.util.logging import configure
+
+try:
+    from sensai.util.logging import configure
+except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)